src/views/Locations.vue

```diff
@@ -151,14 +188,95 @@ export default defineComponent({
       });
       return popover.present();
     },
+    async getFacilities(vSize?: any, vIndex?: any) {
+      const viewSize = vSize ? vSize : process.env.VUE_APP_VIEW_SIZE;
+      const viewIndex = vIndex ? vIndex : '0';
+
+      const payload = {
+        "inputFields": {
+          "facilityId_value": `%${this.queryString}%`,
+          "facilityId_op": "like",
+          "facilityId_ic": "Y",
+          "facilityId_grp": "1",
+          "facilityName_value": `%${this.queryString}%`,
+          "facilityName_op": "like",
+          "facilityName_ic": "Y",
+          "facilityName_grp": "2",
+        } as any,
+        "fieldList": [],
```

Comment:
Pass specific fields needed

Reply:
Passed specific fields in fieldList.

Reply:
Sir, as I have discussed with @dt2patel Sir that we need to show All initially in both filters, if we want to use ion-select and want to show the initial value in UI we need to add the All option in the filters list that's why I have added All in both filter's prop data instead of adding it in response which we are getting from API is this a right approach WDYT please suggest me @adityasharma7 Sir.